cli/src/extension_api.ts

```diff
@@ -26,15 +82,9 @@ export class PhylumApi {
     package_type: string,
     packages: object[],
     project?: string,
-    group?: string,
+    group?: string
```

Comment:
Is this from `prettier`?

I was using `deno fmt` when I made my changes. I guess we'll need to make a choice there.